### PR TITLE
Use const and let instead of var

### PR DESCRIPTION
**Pull request by rabinadk1**
* Used prettier to format all the files
* Tried to convert the function constructors to classes in the `jsbn` folder but it seems like other codes also need to be changed, so didn't do it.

### CODE REVIEW

**Review by bpanthi977 — COMMENTED** [review]
Good point! I should have used const and let instead of var. 
However following chages have to be reverted:

## Indentation
1. I use 8 character tab so your html indendation doesn't look good on mine. 
2. I also use preetier-js for preetifying js but with slightly different arguments 
```lisp
  (setq prettier-js-args '("--trailing-comma" "all"
                           "--bracket-spacing" "false"
                           "--tab-width" "4")))
```

## jsbn
jsbn directory isn't supposed to be modified. (I am sorry that I didn't mention that in README). The reason for this is that any reviewer (from the perspective of verifying the validness of this project for actual election) shouldn't need to review those files. They should be exactly same to the files as provided by the library developer. 

In short, revert back the indendation and formatting changes and keep only const, let, var changes. 
And about the position of <script> tags. I don't see any benefit of moving them to bottom. Loading the html first without the `js` scripts is useless for this case and may lead to usability issue in some slow connection devices.

**Comment by bpanthi977** [comment]
I converted to const and let in 3efb421fa32c2acda70e0207a6c3997d1b3f3466 
Thus closing this pull request for now.
